### PR TITLE
[Feature FIx] Add missing style for wiki

### PR DESCRIPTION
**Pull request by haoyuchen1992**
### Purpose
Wiki has a style missing and panel misses clearfix class.

And also resolved [trello bug](https://trello.com/c/SQlHYxDs/161-text-misaligned-on-settings-page-of-registration-for-retraction)

### Changes
Before Change:
screenshot 2015-07-08 14 50 53
screenshot 2015-07-08 15 09 13

After Change:
screenshot 2015-07-08 15 04 49
screenshot 2015-07-08 15 09 27

### CODE REVIEW

File: website/static/css/pages/wiki-page.css [hunk 1]
```diff
@@ -11,6 +11,15 @@
     padding: 10px;
 }
 
+.osf-panel {
```
Comment (sloria) [inline website/static/css/pages/wiki-page.css]: @haoyuchen1992 This seems to be a more general rule that might be used outside of the wiki page; should this go in style.css?

**Comment by sloria** [comment]
Pass finished. Just one question to address. :factory:

File: website/static/css/pages/wiki-page.css (hunk 1 above)
Comment (haoyuchen1992) [inline website/static/css/pages/wiki-page.css]: Make sense. I searched the code, osf-panel is used in two places. I moved shared css to style.css

Comment (haoyuchen1992) [inline website/static/css/pages/wiki-page.css]: @sloria Don't merge now. I am still checking and make sure it's totally right!

**Pull request merged**